tgui-next/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
```

Comment:
Please fix

Reply:
Fix what exactly

Reply:
This project uses tabs.

You are adding tgui-next to this project, so it too should use tabs.